cpmpy/expressions/globalconstraints.py

```diff
@@ -269,6 +270,10 @@ def value(self):
 
         return pathlen == len(self.args) and idx == 0
 
+
+    def equivalent_decomposition(self):
+        return False
+
     def decompose_negation(self):
```

Comment:
as discussed offline, this too will return a tuple of two expressions (like decompose_comparison below): the first with an expression containing the 'constraint' part; the second with the (total) definitions of any auxiliary variables defined that will be enforced toplevel